src/actions/register.js

```diff
@@ -3,7 +3,7 @@ var { Action, ThunkAction } = require('./types');
 
 function findByUsername(username: string){
     return new Promise((resolve, reject) => {
-        fetch(API_URL + `-public/users/?username=` + username, {
+        fetch(API_URL +`-public/users/?username=` + username, {
```

Comment:
Why is this change here? Don't you use any linter to check your code before commit?